staging/storage/redis/STATEFULSET.md

```diff
@@ -0,0 +1,97 @@
+## Reliable, Scalable Redis on Kubernetes
+
+The following document describes the deployment of a reliable, multi-node Redis on Kubernetes.  It deploys a master with replicated slaves, as well as replicated redis sentinels which are use for health checking and failover.
+
+### Prerequisites
+
+This example assumes that you have a Kubernetes cluster installed and running, and that you have installed the ```kubectl``` command line tool somewhere in your path.  Please see the [getting started](../../../docs/getting-started-guides/) for installation instructions for your platform.
```

Comment:
> ../../../docs/getting-started-guides

these sort of links are all broken in this file. use https://kubernetes.io/ prefix (and follow the redirects).